test/jsonValidationTest.js

```diff
@@ -125,6 +127,16 @@ function validateJsonExistForWorkbook(rootPath, results, file) {
     });
 }
 
+function validateWorkbookExistForSettings(file) {
```

Comment:
we also have a validateSingleWorkbookFile which checks if num of workbooks >1. Should we consolidate these checks for workbooks.length != 1?

Reply:
i don't think so.  because that rule basically runs **iff** it found a workbook in the folder.  not every folder *needs* to have workbooks, so there's no way to make sure this only works for 1 and only one, because then it would fail on other folders that don't have workbooks.

this one makes sure nobody puts more than one in a folder.

the new test makes sure if there's a settings.json, there's also exactly one workbook.

i could *probably* consolidate all these rules (there's a separate one for if you have a workbook it also finds settings.json or category resources, but at this point i'd like to have small simple obvious checks that might be redundant instead of a single big check that does all the things?